test/provider/ContextHandler.unittest.ts

```diff
@@ -27,29 +31,34 @@ function createMockAppWindowWithName(name: string): AppWindow {
     return mockAppWindow;
 }
 
-function setChannelModelToProvideWindows(...windows: AppWindow[]): void {
+function setGetChannelMembersToReturn(...windows: AppWindow[]): void {
```

Comment:
Is this equivalent to `mockGetChannelMembers.mockReturnValue(windows);`?

Reply:
Yes.